app/js/const-service.js

```diff
@@ -48,12 +50,13 @@ angular.module('cttvServices').
                 CHEMBL :              "/data_sources#chembl",
                 DISGENET :            "/data_sources",  // no longer used
                 EPMC :                "/data_sources#text_mining",
+                NFERX :                "/data_sources#nferx",
                 EVA_SOMATIC :         "/data_sources#eva",
                 INTOGEN :             "/data_sources#intogen",
                 GENE_2_PHENOTYPE :    "/data_sources#gene2phenotype"
             },
             datatypesOrder: [
-                "GENETIC_ASSOCIATION", "SOMATIC_MUTATION", "KNOWN_DRUG", "AFFECTED_PATHWAY", "RNA_EXPRESSION", "LITERATURE", "ANIMAL_MODEL"
+                "GENETIC_ASSOCIATION", "SOMATIC_MUTATION", "KNOWN_DRUG", "AFFECTED_PATHWAY", "RNA_EXPRESSION", "LITERATURE", "LITERATURE_NFERX","ANIMAL_MODEL"
```

Comment:
are you sure you want a new datatype for this? looks like a new datasource for the literature datatype?